webdriver/support/http_request.py

```diff
@@ -1,6 +1,7 @@
 import contextlib
 import httplib
 
+
```

Comment:
Well, I do love conformance. In the future, a dedicated commit (even for a one-line change) would be helpful in communicating your intentions. But it's no big deal :)